### DISCUSSION TITLE
Repository Order

### DISCUSSION THREAD

**Discussion by RobinLe0** [community/community#141320] [Mobile] [answered]
On Android, the repository list has no logical order. It's not ordered by last commit, it's not ordered alphabetically. Maybe there should be an option to sort the repo list?
I have many repos and it's a horrible experience to find one. I have to go through every single entry to find what I'm looking for.

**Comment by alcere** [comment]
Hi @RobinLe1402. Thank you so much for taking the time to share your feedback with us! The issue you're experiencing is occurring within the repository section of your Profile tab or in the Home tab? Or maybe in both?

**Reply by RobinLe0** [reply]
I can order it under Profile > Repositories, but not on Home > My work > Repositories.

**Reply by alcere** [reply]
I see now! Just in case it helps the Repositories displayed under My Work -> Repositories are sorted by the push time, being on top of the repository you pushed changes last. But I totally see how this behaviour is totally hidden. I will open a feature request to allow users change the sort option for this screen and the team will consider this work for future releases. Thanks for your feedback!

**Reply by RobinLe0** [reply]
But I don't think that that applies to my repos. Only maybe if it's actually sorted by time of day.
I know exactly which three or four repos I last touched, last time yesterday.
But the top repository is none of them. In fact, I didn't touch that repo in months!

**Reply by alcere** [reply, marked as answer]
You are right @RobinLe1402. I was able to reproduce the problem. I have created an internal issue, and the team will look into this issue. Thanks for reporting!
